graphistry/outliers.py

```diff
@@ -0,0 +1,173 @@
+from typing import Any, List, Optional, Union
+
+import matplotlib.font_manager
+import matplotlib.pyplot as plt
```

Comment:
are methods in `outliers.py` lazily imported or always-on top-level imports? if not, defer import to use b/c matplotlib dependency is optional

probably less fragile if we make just-in-time imports either way..